test/CMakeLists.txt

```diff
@@ -290,7 +290,7 @@ foreach(SDK ${SWIFT_SDKS})
           set(test_mode_target_suffix "-${test_mode}")
         endif()
 
-        add_custom_target("check-swift${test_subset_target_suffix}${test_mode_target_suffix}${VARIANT_SUFFIX}"
+        add_custom_target("check-swift${test_subset_target_suffix}${test_mode_target_suffix}${VARIANT_SUFFIX}-default"
```

Comment:
Is there any particular reason we need to suffix these names with `-default`? Why not just leave them alone and only add the `-custom` targets?

Reply:
Not really, I just though that it might make sense to split into "default" behavior with statically added directories and "custom" when caller can add arbitrary directories. Will make it so existing target is not renamed.